test/EventuallyConsistentTestTraitTest.php

```diff
@@ -58,4 +58,20 @@ public function testCatchAllExceptionsTest()
         $this->runEventuallyConsistentTest($func, $retries, true);
         $this->assertEquals($i, $retries);
     }
+
+    public function testCatchAllExceptionsWithInstanceVarTest()
+    {
+        $retries = 4;
+        $i = 0;
+        $func = function () use (&$i, $retries) {
+            if (++$i == $retries) {
+                // return on the final retry
+                return;
+            }
+            throw new \Exception('Something goes wrong');
+        };
+        $this->catchAllExceptions = true;
```

Comment:
In the future, this could mess up other tests if it's not set back to the default in a setup/teardown block.

In fact, we may also want to have a few tests with `@expectedException \Exception` for when this value is false and the option is not set for `runEventuallyConsistentTest`